emsdk.py

```diff
@@ -2672,6 +2672,9 @@ def expand_sdk_name(name):
           backend = 'upstream'
         else:
           backend = 'fastcomp'
+      if backend == 'fastcomp' and version_key(fullname) >= (2, 0, 0):
+        print('Note: fastcomp is not longer available in version 2.0.0 and above')
+        return name
```

Comment:
it might be safer to error here? both code-wise it would be clearer that everything halts here. and the error for users would be better I think.

Reply:
The error this produces seems pretty good to me:

```
$ ./emsdk install 2.0.0-fastcomp
Note: fastcomp is not longer available in version 2.0.0 and above.
Error: No tool or SDK found by name '2.0.0-fastcomp'.
```

Basically we are failing expand the input given by the user and then the calling code displays the same error the would see if they had made any other kind of typo or something.

Reply:
I don't love the fact that it depends on `2.0.0-fastcomp` not existing and being errored on elsewhere, when we know this should fail right here. But that's a reasonable error.